src/test/regress/sql/explain_analyze.sql

```diff
@@ -37,3 +47,24 @@ select gp_inject_fault('explain_analyze_sort_error', 'reset', dbid)
     from gp_segment_configuration where role = 'p' and content > -1;
 drop table sort_error_test1;
 drop table sort_error_test2;
+
+--
+-- Test correct slice stats reporting in case of duplicated subplans
+--
+
+create table slice_test(i int, j int) distributed by (i);
+
+-- explain_processing_off
+-- create duplicate subplan in QE slice
+explain (analyze, timing off, costs off) select a.i from slice_test a
```

Comment:
Is it really a good idea to do `explain analyze` without data?

Reply:
The issue was reproducible without any data. I could add some data to it if necessary though.

Reply:
Add it if you don't mind.